e2e/fixtures.ts

```diff
@@ -79,22 +99,47 @@ export async function loadApp(page: Page, entities?: Entity[]) {
  * This implementation is not fully compatible with serialization in the app.
  */
 function serializeEntities(entities: Entity[]): unknown[] {
-  const entitySchemaService = new EntitySchemaTransformer([
-    new DefaultDatatype(),
-    new StringDatatype(),
-    new DateWithAgeDatatype(),
-    new DateOnlyDatatype(),
-    new LongTextDatatype(),
-
-    // the following datatypes have dependencies through Angular DI - need workarounds in case they are needed for test data
-    //new ConfigurableEnumDatatype(null),
-    //new LocationDatatype(null),
-    //new EntityDatatype(null, null, null),
-    //new EventAttendanceMapDatatype(entitySchemaService),
-  ]);
+  const injector = Injector.create({
```

Comment:
ah, nice. This looks easier than I would've thought :slightly_smiling_face: 

Then we could also undo the split between EntitySchemaTransformer and EntitySchemaService that I previously added to disentangle the part requiring DI. I think having those as two classes is only confusing, if it is not absolutely necessary